eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/location/LocationSelectionFragment.java

```diff
@@ -71,25 +77,34 @@ public static Fragment newInstance()
     public void onCreate(@Nullable Bundle savedInstanceState)
     {
         super.onCreate(savedInstanceState);
-        setRetainInstance(true);
-
-        mGoogleApiClient = new GoogleApiClient
-                .Builder(getContext().getApplicationContext())
-                .addApi(Places.GEO_DATA_API)
-                .addApi(Places.PLACE_DETECTION_API)
-                .build();
-        /**
-         * TODO enableAutomanage OR manual error handling
-         * enableAutoManage() on the builder would enable automatic default error handling as well,
-         * but it's tricky to get initialization correctly with Activity and retained Fragment lifecycles.
-         * Either enable automanage or add 'manual' error handling with addOnConnectionFailedListener().
-         *
-         * See https://developers.google.com/android/reference/com/google/android/gms/common/api/GoogleApiClient.Builder.html#enableAutoManage(android.support.v4.app.FragmentActivity, com.google.android.gms.common.api.GoogleApiClient.OnConnectionFailedListener)
-         */
-        mGoogleApiClient.connect();
-
-        mSuggestionItems = new BasicChangeableListItems();
-        mPlaceListController = new PlaceListControllerImpl(mGoogleApiClient, this, mSuggestionItems);
+
+        mSearchListItems = new UpdateDelaying(new BasicSearchListItems(), 10, 400);
```

Comment:
It's probably better to use `getResources().getInteger(android.R.integer.config_mediumAnimTime)` instead of a hard coded `400`.